src/calendar.scss

```diff
@@ -3,178 +3,254 @@
+$fd-calendar-width: $fd-calendar-item-width * 8 + 2 * $fd-calendar-padding !default;
+
+// Compact dimensions
+$fd-calendar-padding-compact: 0.5rem !default;
+$fd-calendar-item-width-compact: 2rem !default;
+$fd-calendar-item-height-compact: 2rem !default;
+$fd-calendar-width-compact: $fd-calendar-item-width-compact * 8 + 2 * $fd-calendar-padding-compact !default;
+
+// Mobile dimensions
+$fd-calendar-button-height-mobile: 2.75rem !default;
+$fd-calendar-padding-mobile-landscape: 0 !default;
+$fd-calendar-padding-mobile-portrait: 1rem 0 0 0 !default;
+$fd-calendar-item-helper-height-landscape: 1.625rem !default;
+
+// Calendar items variations
+$fd-calendar-item-color: var(--sapTextColor) !default;
+$fd-calendar-item-background: #f7f7f7 !default; // Should be darken(var(--sapList_Background), 3)
+$fd-calendar-item-background-hover: #d9d9d9 !default; // Should be darken(var(--sapList_Background), 15)
+
+$fd-calendar-item-helper-height: 2rem !default;
+$fd-calendar-item-helper-font: var(--sapFontSmallSize) !default;
+$fd-calendar-item-helper-color: var(--sapContent_LabelColor) !default;
+
+$fd-calendar-item-active-color: var(--sapContent_ContrastTextColor) !default;
+$fd-calendar-item-active-background: var(--sapActiveColor) !default;
+$fd-calendar-item-active-background-hover: #095caf !default; // Should be lighten(--sapActiveColor, 3)
+$fd-calendar-item-active-inner-border-radius: 0.125rem !default; // Should be lighten(--sapList_BorderColor,10)
+$fd-calendar-item-active-inner-border-width: 0.0625rem !default; // Should be lighten(--sapList_BorderColor,10)
+$fd-calendar-item-active-inner-border-color: #ffffff !default; // Should be lighten(--sapList_BorderColor,10)
+$fd-calendar-item-active-inner-border: $fd-calendar-item-active-inner-border-width solid $fd-calendar-item-active-inner-border-color !default;
+
+$fd-calendar-item-inactive-color: var(--sapContent_LabelColor) !default;
+$fd-calendar-item-inactive-background: var(--sapList_Background) !default;
+$fd-calendar-item-inactive-background-hover: #e6e6e6 !default; // Should be darken(var(--sapList_Background), 10)
+
+$fd-calendar-item-focused-outline-offset: -0.1875rem !default;
+$fd-calendar-item-focused-outline-color: var(--sapContent_FocusColor) !default;
+$fd-calendar-item-focused-outline-width: var(--sapContent_FocusWidth) !default;
+$fd-calendar-item-focused-outline-style: var(--sapContent_FocusStyle) !default;
+
+$fd-calendar-item-weekend-background: #dedede !default; // darken(--sapList_Background, 13)
+$fd-calendar-item-weekend-background-hover: #cccccc !default; // darken(--sapList_Background, 20)
+
+$fd-calendar-item-range-background: var(--sapList_SelectionBackgroundColor) !default;
+$fd-calendar-item-range-background-hover: var(--sapList_Hover_SelectionBackground) !default;
+$fd-calendar-item-range-border: solid 0.0625rem var(--sapList_SelectionBorderColor) !default;
+
+$fd-calendar-item-current-border: solid medium var(--sapAccentColor4) !default;
```

Comment:
Unfortunately meduim doesn't match the specs, we go back to `0.125rem` 
example:
![image](https://user-images.githubusercontent.com/26483208/73268379-33261b00-41db-11ea-92b1-d128df8e9e4d.png)
Review:
![image](https://user-images.githubusercontent.com/26483208/73268424-3e794680-41db-11ea-8fa9-bedb2c82bd78.png)